saleor/dashboard/widgets.py

```diff
@@ -17,7 +16,7 @@ def __init__(self, attrs=None):
 class MoneyRangeWidget(widgets.RangeWidget):
     def __init__(self, attrs=None):
         self.currency = getattr(settings, "DEFAULT_CURRENCY")
```

Comment:
Is it expected that you take currency from settings here?

Reply:
The issue here is that we don't really know what currency we want to filter the objects by. So we assume it is the default currency until we drop dashboard 1.0 and implement multi-currency support.